crates/winmd/src/types/type_name.rs

```diff
@@ -123,7 +123,15 @@ impl TypeName {
         };
 
         quote! {
-            #format
+            static mut VAL: &'static str = "";
```

Comment:
Are you sure this works? I believe the static storage is shared by all specializations of the generic. So `IVector<i32>` will have the same signature as `IVector<f32>`. This is unlike in C++ where each specialization gets its own static storage.

Reply:
Ach sorry. You're right. I was wondering why it was so simple this time around.